src/cli/cli_srp_server.cpp

```diff
@@ -75,32 +95,51 @@ template <> otError SrpServer::Process<Cmd("addrmode")>(Arg aArgs[])
 }
 
 #if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+
 /**
- * @cli srp server auto
+ * @cli srp server auto (enable,disable)
+ * @code
+ * srp server auto enable
+ * Done
+ * @endcode
  * @code
  * srp server auto
- * Disabled
+ * Enabled
  * Done
  * @endcode
- * @par api_copy
- * #otSrpServerIsAutoEnableMode
+ * @cparam srp server auto [@ca{enable}|@ca{disable}]
+ * @par
+ * Enables or disables the auto-enable mode on the SRP server.
+ * @par
+ * When this mode is enabled, the Border Routing Manager controls if and when
```

Comment:
Can we note somewhere that this command requires OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE?  None of the others do so we should call it out for this one.